Sources/UIView.swift

```diff
@@ -206,7 +206,13 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
     /// Adds a subview without touching the view's layer or any of its sublayers.
     /// We need to be able to add layers at an index that isn't directly related to its subview index.
     private func insertSubviewWithoutTouchingLayer(_ view: UIView, at index: Int) {
-        if view.superview != nil { view.removeFromSuperview() }
```

Comment:
`removeFromSuperview` called `layer.removeFromSuperlayer()`, which we do not want here.